DQMServices/Core/python/DQMEDHarvester.py

```diff
@@ -0,0 +1 @@
+from FWCore.ParameterSet.Config import EDProducer as DQMEDHarvester
```

Comment:
Why not
```python
import FWCore.ParameterSet.Config as cms
cms.DQMEDHarvester = cms.EDProducer
```
so that the user code can use a more familiar syntax:
```python
import FWCore.ParameterSet.Config as cms
import DQMServices.Core.DQMEDHarvester
 
SiStripGainsPCLHarvester = cms.DQMEDHarvester( ... )
```